spec/features/process_a_request_spec.rb

```diff
@@ -3,13 +3,22 @@
 RSpec.feature 'Processing a request', js: true do
   include ActiveJobHelper
 
-  let(:prison) { create(:prison, name: 'Reading Gaol') }
-  let(:email_address) { 'visitor@test.example.com' }
+  let(:visitor_email_address) { 'visitor@test.example.com' }
+  let(:prison_email_address) { 'prison@test.example.com' }
+  let(:prison) {
+    create(
+      :prison,
+      name: 'Reading Gaol',
+      email_address: prison_email_address
+    )
+  }
   let(:vst) {
```

Comment:
More seriously, `visit` is already taken by Capybara for visiting a page. An unfortunate clash.